packages/fe-fpm-writer/src/column/index.ts

```diff
@@ -0,0 +1,107 @@
+import { validateVersion, validateBasePath } from '../common/validate';
+import { create as createStorage } from 'mem-fs';
+import { create, Editor } from 'mem-fs-editor';
+import { CustomTableColumn, InternalCustomTableColumn } from './types';
+import { join } from 'path';
+import { render } from 'ejs';
+import { Manifest } from '../common/types';
+import { setCommonDefaults } from '../common/defaults';
+
+/**
+ * Get the template folder for the given UI5 version.
+ *
+ * @param ui5Version required UI5 version.
+ * @returns path to the template folder containing the manifest.json ejs template
+ */
+export function getManifestRoot(ui5Version?: number): string {
+    if (ui5Version === undefined || ui5Version >= 1.86) {
+        return join(__dirname, '../../templates/column/1.86');
+    } else if (ui5Version === 1.85) {
+        return join(__dirname, '../../templates/column/1.85');
+    } else {
+        return join(__dirname, '../../templates/column/1.84');
+    }
```

Comment:
For the column case, we could do the versioning slightly different and remove the duplications, however, in discussions with @stefanschreck we agreed that we want to try to keep the version specific templates separate so that they don't accidentally break when changing the template for the latest UI5 version i.e. once committed these old-version-templates shall not be touched anymore.
But, to be honest, I am not to sure yet, whether this is the best approach for versioning. Can we do it like this for now and change the approach if we have a better idea?

Reply:
Sure 👍